### ISSUE TITLE
UI is broken after upgrading to polkadot v0.9.11

### ISSUE THREAD

**Issue by GopherJ** [polkadot-js/apps#6349]
<img width="1387" alt="image" src="https://user-images.githubusercontent.com/33961674/138068584-0de6f6b0-72b7-401c-ab3a-5142cd803805.png">

**Comment by GopherJ** [comment]
cannot choose account  it seems, and the `gray` text looks strange

**Comment by jacogr** [comment]
The input there is a `[u8; 32]` as specified by the metadata in v14. There is no hint in the metadata that indicates that this is a publicKey. Nothing broken, it is expected - the API doesn't assign types anymore, rather the raw types and structures are supplied by the metadata.

**Comment by GopherJ** [comment]
yes it's correct but just hard to use, also we don't have downward transfer I believe

**Comment by GopherJ** [comment]
![Uploading image.png…]()

having this now

**Comment by GopherJ** [comment]
@jacogr what shall I do? I'm not yet very clear on this. The westend is also v0.9.11

**Comment by jacogr** [comment]
Westend has broken weights for xcm. If you try and do a teleport, you will see the same.
